Assets/MRTK/Core/Providers/InputSimulation/InputSimulationService.cs

```diff
@@ -134,7 +134,20 @@ public void ResetHandRight()
         /// <summary>
         /// If true then camera forward direction is used to simulate eye tracking data.    
         /// </summary>
-        public bool SimulateEyePosition { get; set; }
+        [Obsolete("Check the EyeGazeSimulationMode instead")]
+        public bool SimulateEyePosition => eyeGazeSimulationMode != EyeGazeSimulationMode.Disabled;
```

Comment:
Removing the setter here is a breaking change. Can we do this in a different way, where this can still be the getter but that we don't break anybody who was previously setting this in code?

Reply:
Probably just
`set { eyeGazeSimulationMode = value ? EyeGazeSimulationMode.CameraForwardAxis : EyeGazeSimulationMode.Disabled; }`, which should maintain backwards compatibility and not change behavior.